Library/Homebrew/utils/formatter.rb

```diff
@@ -108,4 +108,11 @@ def pluralize(count, singular, plural = nil, show_count: true)
 
     show_count ? "#{count} #{words}" : words
   end
+
+  def enumeration(*items)
```

Comment:
`array_comma_and` or something to me would read a bit more obviously; I was a bit confused as to what `enumeration` meant.